### PR TITLE
Fixing broken tests by fixing the package find.

### PR DESCRIPTION
**Pull request by toddm**
@TimBarham

### CODE REVIEW

**Comment by TimBarham** [comment]
:shipit:!

**Comment by joelverhagen** [comment]
Can haz targeted unit tests for this class? This seems like a slam dunk though.

File: test/NuGet.Core.Tests/NuGet.Protocol.Core.v3.Tests/LocalV2FindPackageByIdResourceTests.cs [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+namespace NuGet.Protocol.Core.v3.Tests
+{
+    public class LocalV2FindPackageByIdResourceTests
+    {
+        [Fact]
+        public async Task LocalV2FindPackageByIdResource_LocalSource()
+        {
+            // Arrange
+            var workingDirectory = TestFileSystemUtility.CreateRandomTestFolder();
```
Comment (joelverhagen) [inline test/NuGet.Core.Tests/NuGet.Protocol.Core.v3.Tests/LocalV2FindPackageByIdResourceTests.cs]: using to clean up the test dir

**Comment by joelverhagen** [comment]
:shipit:

**Pull request merged**